lib/rets/parser/compact.rb

```diff
@@ -6,7 +6,7 @@ class Compact
       InvalidDelimiter = Class.new(ArgumentError)
 
       def self.parse_document(xml)
-        doc = Nokogiri.parse(xml.to_s)
+        doc = Nokogiri.parse(CGI.unescapeHTML(xml.to_s))
```

Comment:
This actually breaks the xml parsing in at least one of our mlses. I'm going to revert and look at pushing this fix higher up, after the xml has been parsed.